src/pages/templates/pages/partial/dashboard_panel_popular_reads_student_messages.html

```diff
@@ -0,0 +1,34 @@
+{% if data.view == 'assigned' %}
+    {% if data.all|length == 0 %}
+        Here’s where you’ll see readings assigned by your teacher.
+    {% else %}
+        These are your {{ data.all|length }} most recently assigned readings. See all assigned
```

Comment:
Currently says "These are your 1 most..." if there is only one.
But perhaps rather than do the singular/plural thing we can drop this message/link entirely if there are less than 4 readings? It becomes useless.

Reply:
Part of the problem is that the readings lists are truncated to 3 readings in the code, so there is always fewer than 4 readings by the time the template is processed.  I've added a new property, `total`, that is used by the template.

Also, experimenting with removing the link depending on that total, I found (1) the experience to be inconsistent ("where did the link go?"), and (2) the link was still convenient for navigating to the library showing the assigned readings.  So, I left the message/link in even when there was fewer than 4 items in the total.  But, that is a sample size of one ...  Anyhow, if there are fewer than four, the message leaves out the "3":  "These are your most recently assigned readings ...".